### ISSUE TITLE
batch norm layers

### ISSUE THREAD

**Issue by 1292765944** [Eniac-Xie/faster-rcnn-resnet#12]
I don't find any difference in batchnorm layer between Microsoft's caffe and official caffe. How does the Microsoft's caffe reduces the memory usage?

Best!
